### DISCUSSION TITLE
How to handle failures with Mutiny

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#30028] [Quarkus Blog/Website]
# How to handle failures with Mutiny

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/mutiny-failure-handling/

**Comment by ashutoshk2206** [comment]
I am trying to understand the difference between call() and tranformToUni(). Above example works fine but when I change "transformToUniAndConcatenate" to "call", it doesn't work as expected.
Can you please share some clarification on when to use call vs transform ?

**Reply by ichrist-gr** [reply]
With .call() operator you declare that you want to perform a non-blocking action and you actually do not care about it's returned value. With .transformToUni() operator you execute a non-blocking operation and you want to retrieve the response from this operation. So you get as a result the response from your db/rest-service
